tests/tracer/test_propagation.py

```diff
@@ -456,15 +495,22 @@ def test_get_wsgi_header(tracer):
 
 # for testing with other propagation styles
 TRACECONTEXT_HEADERS_VALID_BASIC = {
-    _HTTP_HEADER_TRACEPARENT: "00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01",
+    _HTTP_HEADER_TRACEPARENT: "00-80f198ee56343ba864fe8b2a57d3eff7-00f067aa0ba902b7-01",
     _HTTP_HEADER_TRACESTATE: "dd=s:2;o:rum",
 }
 
 TRACECONTEXT_HEADERS_VALID = {
-    _HTTP_HEADER_TRACEPARENT: "00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01",
+    _HTTP_HEADER_TRACEPARENT: "00-80f198ee56343ba864fe8b2a57d3eff7-00f067aa0ba902b7-01",
     _HTTP_HEADER_TRACESTATE: "dd=s:2;o:rum;t.dm:-4;t.usr.id:baz64,congo=t61rcWkgMzE",
 }
 
+if asbool(os.getenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED")):
```

Comment:
I think I'm missing something, how does putting this here affect the tests? Doesn't this code only run once since it's not in any method or parametrize?

Reply:
This is a bit subtle. We run the whole tracer suite with [DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=True and DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=False](https://github.com/DataDog/dd-trace-py/blob/1.x/riotfile.py#L280). This ensures all tracer tests are compatible with 128bit trace ids. When 128bit trace ids are enabled by default (hopefully this happens by v1.13.0), I will remove this env var from our riot configurations.

Reply:
Ah I missed that, understood!